CHANGELOG.md

```diff
@@ -8,6 +8,15 @@ The format is based on [these versioning and changelog guidelines](https://git.i
 
 ---
 
+## 5.9.0 - 2020-10-22
+
+### Bug fixes
+
+- Fixed `Filters` overflow ([#3532](https://github.com/Shopify/polaris-react/pull/3532))
+- Updated Textfield with a type of number to not render a spinner if step is set to 0 ([#3477](https://github.com/Shopify/polaris-react/pull/3477))
+- Refactored `Portal` to render all `Portals` in a single container ([#3544](https://github.com/Shopify/polaris-react/pull/3544))
```

Comment:
Should this and the TextField change be considered enhancements instead of bug fixes?